imports/api/server/publishFunctions.js

```diff
@@ -25,24 +25,29 @@ export const getBestQualityMatch = function (animeDetailsArray) {
         return prev.sigma > curr.sigma ? prev : curr;
     });
 
-    // we define most interesting chance of winning to be the one closest to 0.5, thus start with 1 which is one of the 2 possible values for least interesting (0 being the other)
-    mostInterestingChanceOfWinning = 1;
-    bestQualityMatchObjectIndex = 0;
+    // define match quality as in equation (7) of https://www.microsoft.com/en-us/research/publication/trueskilltm-a-bayesian-skill-rating-system/
+    matchQualityFunction = function(defaultSigma, mu1, mu2, sigma1, sigma2) {
```

Comment:
Also, I realise it might be hard to test with crazy decimal values, but there should be some easy inputs for which we can verify the function is working as expected. Could you mention a few of them so that I can add them as a test?